packages/ui/src/loading/LoadingNotification.tsx

```diff
@@ -1,3 +1,4 @@
+// TODO: Remove this component after May, 2023
```

Comment:
Why May?

Reply:
I'm not sure whether any BCapp functional tests or out side apps use it, so leave it in place for an easy rollback. 
Thought two months should be enough. 😂
Will run functional tests as well when I have addressed all comments.

Reply:
I removed `LoadingNotification.tsx` now.